### PR TITLE
Choose specific host label to deploy the MongoDB cluster..

### PR DESCRIPTION
**Pull request by suizman**
Adds the ability to choose specific host label to deploy the MongoDB cluster..

### CODE REVIEW

File: templates/MongoDB/2/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  environment:
+    MONGO_SERVICE_NAME: mongo-cluster
+  tty: true
+  entrypoint: /opt/rancher/bin/entrypoint.sh
+  command:
+  - --replSet
+  - "${replset_name}"
+  image: mongo:3.2
+  labels:
+    io.rancher.scheduler.affinity: "${host_label}"
```
Comment (rawmind0) [inline templates/MongoDB/2/docker-compose.yml]: Line 12 are wrong...It has to be...
    io.rancher.scheduler.affinity:host_label: ${host_label}

File: templates/MongoDB/2/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    io.rancher.sidekicks: mongo-base, mongo-datavolume
+  volumes_from:
+    - mongo-datavolume
+    - mongo-base
+mongo-base:
+  restart: always
+  net: none
+  tty: true
+  labels:
+    io.rancher.scheduler.affinity: "${host_label}"
```
Comment (rawmind0) [inline templates/MongoDB/2/docker-compose.yml]: Line 23...
--    io.rancher.scheduler.affinity: "${host_label}"
++    io.rancher.scheduler.affinity:host_label: ${host_label}

**Comment by rawmind0** [comment]
Hi @suizman...

thanks for the PR and contribute....

Could you please review the annotations and change them??

Best regards....

**Comment by suizman** [comment]
Tnx @rawmind0, it's fixed now

**Comment by rawmind0** [comment]
LGTM...

Thanks @suizman

**Pull request merged**